src/audit-logs/interfaces/audit-log-event-options.interface.ts

```diff
@@ -0,0 +1,26 @@
+export interface AuditLogActor {
+  id: string;
+  name?: string;
+  type: string;
+  metadata?: Record<string, string | number | boolean>;
+}
+
+export interface AuditLogTarget {
+  id: string;
+  name?: string;
+  type: string;
+  metadata?: Record<string, string | number | boolean>;
+}
+
+export interface AuditLogEventOptions {
```

Comment:
```suggestion
export interface AuditLogCreateEventOptions {
```

Reply:
Or if this is the shape of the object itself, perhaps it should just be `AuditLogEvent` and `AuditLogCreateEv evntOptions` would house just the options?